lib/rentlinx/models/unit.rb

```diff
@@ -10,7 +10,7 @@ class Unit < Base
                   :bedrooms, :fullBaths, :halfBaths, :isMobilityAccessible,
                   :isVisionAccessible, :isHearingAccessible, :rentIsBasedOnIncome,
                   :isOpenToLease, :dateAvailable, :dateLeasedThrough, :numUnits,
-                  :numAvailable].freeze
+                  :numAvailable, :address].freeze
```

Comment:
It's designed to be what we call `address1` and `address2` in property. For example "123 Fake St. Unit 13"